src/runtime/node/os/index.ts

```diff
@@ -0,0 +1,60 @@
+import type os from "node:os";
+import { notImplemented } from "../../_internal/utils";
+import mock from "../../mock/proxy";
+
+export const arch: typeof os.arch = notImplemented("os.arch");
+export const availableParallelism: typeof os.availableParallelism =
+  notImplemented("os.availableParallelism");
+export const cpus: typeof os.cpus = notImplemented("os.cpus");
```

Comment:
Some simple items can be simply non-throwing and return a  fallback value IMO. It increases compatibility (end-users don't have a choice when they need a lib that relyins on these APIs. If it throws it does not fixes anything but fallback value would)

Reply:
hmmm... do you have a suggested value here?

When I test this, the output on my laptop is not trivial:

```
> require('os').cpus()
[
  {
    model: 'Apple M1 Pro',
    speed: 2400,
    times: {
      user: 148364570,
      nice: 0,
      sys: 97148930,
      idle: 304228910,
      irq: 0
    }
  },
  {
    model: 'Apple M1 Pro',
    speed: 2400,
    times: {
      user: 147962800,
      nice: 0,
      sys: 95743880,
      idle: 309692400,
      irq: 0
    }
  },
  {
    model: 'Apple M1 Pro',
    speed: 2400,
    times: {
      user: 134111910,
      nice: 0,
      sys: 35374800,
      idle: 402528560,
      irq: 0
    }
  },
  {
    model: 'Apple M1 Pro',
    speed: 2400,
    times: { user: 84991730, nice: 0, sys: 25427380, idle: 463970630, irq: 0 }
  },
  {
    model: 'Apple M1 Pro',
    speed: 2400,
    times: { user: 55427660, nice: 0, sys: 14547330, idle: 506192030, irq: 0 }
  },
  {
    model: 'Apple M1 Pro',
    speed: 2400,
    times: { user: 39030960, nice: 0, sys: 10133530, idle: 527638760, irq: 0 }
  },
  {
    model: 'Apple M1 Pro',
    speed: 2400,
    times: { user: 40421640, nice: 0, sys: 6451750, idle: 530715920, irq: 0 }
  },
  {
    model: 'Apple M1 Pro',
    speed: 2400,
    times: { user: 19716300, nice: 0, sys: 3817220, idle: 554515890, irq: 0 }
  },
  {
    model: 'Apple M1 Pro',
    speed: 2400,
    times: { user: 12116580, nice: 0, sys: 2508300, idle: 563667690, irq: 0 }
  },
  {
    model: 'Apple M1 Pro',
    speed: 2400,
    times: { user: 9789590, nice: 0, sys: 2036340, idle: 566538860, irq: 0 }
  }
]
```

Since this change is not making the situation worse than it is now with the current module level mock, I suggest we merge this as is to increase the "presence" coverage, and then do a second pass and replace these throwing functions with some mock value.

In general, our strategy is to get to 100% of API "presence" coverage quickly, and then do a second pass of filling in the implementation where it's simple or important for real apps/libraries.

Reply:
please note that currently all missing exports _are_ polyfilled (via proxy) but won't throw an error. It can potentially (and in case of this module i suspect more likely) that we break current projects because the polyfill changes from noop to error throws.

[stackblitz for example](https://stackblitz.com/edit/stackblitz-starters-ugx7ss?description=Starter%20project%20for%20Node.js,%20a%20JavaScript%20runtime%20built%20on%20Chrome%27s%20V8%20JavaScript%20engine&file=index.js,package.json&title=node.new%20Starter) returns random 8 cores.